bundles/org.openhab.binding.melcloud/src/main/java/org/openhab/binding/melcloud/internal/handler/MelCloudAccountHandler.java

```diff
@@ -207,7 +207,7 @@ private void startConnectionCheck() {
                     }
                 }
             };
-            connectionCheckTask = scheduler.scheduleWithFixedDelay(runnable, 0, 60, TimeUnit.SECONDS);
+            connectionCheckTask = scheduler.scheduleWithFixedDelay(runnable, 0, 300, TimeUnit.SECONDS);
```

Comment:
Is this change needed? As checking if it is connected or not is not actually causing an API-call, so i guess it should not be affected by the new API limits.

Reply:
I changed it as a safety measure as the check itself doesn't perform an API call when the connection is still open but when there's an issue logging in it will trigger an API call every X seconds.
This value being originally 60 will cause the "excessive traffic" message on top of the login issue.
Once they mark your IP as generating excessive traffic, you need to stop connecting for a few hours to reset.
Changing to 300 should prevent that.